crates/nodeapi/src/lib.rs

```diff
@@ -6,10 +6,10 @@
 
 pub mod error;
 
-use ntex::web::{self, App, HttpResponse, HttpServer, Responder};
-
 use error::NodeApiError;
+use ntex::web::{self, App, HttpResponse, HttpServer, Responder};
 use serde::{Deserialize, Serialize};
+use std::sync::{Arc, Mutex};
```

Comment:
Let's use `tokio::sync`

Reply:
we use std::Mutex because of this: https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use